pkg/client/client.go

```diff
@@ -322,6 +322,20 @@ func FormatStatusResponse(w io.Writer, sr *models.StatusResponse, sd StatusDetai
 		fmt.Fprintf(w, "\n")
 	}
 
+	if sr.Srv6 != nil {
+		var fields []string
+
+		status := "Disabled"
+		fields = append(fields, status)
+
+		if sr.Srv6.Enabled {
+			fields[0] = "Enabled"
+			fields = append(fields, fmt.Sprintf("SRv6EncapMode:\t%s", sr.Srv6.Srv6EncapMode))
```

Comment:
This will become an output like this right?

```
SRv6: Enabled   SRv6EncapMode:   reduced
```

It's just a small cosmetic suggestion, but I prefer surrounding encap mode into braces and making it lower case as we do for KubeProxyReplacement. With the current way, `SRv6` and `SRv6EncapMode` look like an independent status while in fact, SRv6EncapMode is a "sub" status of SRv6. 

```
KubeProxyReplacement: False   [eth0 , net0  (Direct Routing)]
SRv6: Enabled   [encap-mode: reduced]
```

Reply:
Yes, it was printing like this `SRv6: Enabled   SRv6EncapMode:   reduced`.

I updated as requested, the output is like this now:

```
# cilium status
KVStore:                 Ok   Disabled
Kubernetes:              Ok   1.26 (v1.26.3) [linux/arm64]
Kubernetes APIs:         ["EndpointSliceOrEndpoint", "cilium/v2::CiliumClusterwideNetworkPolicy", "cilium/v2::CiliumEndpoint", "cilium/v2::CiliumNetworkPolicy", "cilium/v2::CiliumNode", "cilium/v2alpha1::CiliumCIDRGroup", "core/v1::Namespace", "core/v1::Pods", "core/v1::Service", "networking.k8s.io/v1::NetworkPolicy"]
KubeProxyReplacement:    Disabled   
Host firewall:           Disabled
SRv6:                    Enabled   [encap-mode: reduced]
CNI Chaining:            none
Cilium:                  Ok   1.15.0-dev (v1.15.0-dev-22bee4a1d7)
NodeMonitor:             Listening for events on 5 CPUs with 64x4096 of shared memory
Cilium health daemon:    Ok   
IPAM:                    IPv4: 2/254 allocated from 10.244.0.0/24, IPv6: 2/18446744073709551614 allocated from fd00:10:244::/64
IPv4 BIG TCP:            Disabled
IPv6 BIG TCP:            Disabled
BandwidthManager:        Disabled
Host Routing:            Legacy
Masquerading:            IPTables [IPv4: Enabled, IPv6: Enabled]
Controller Status:       20/20 healthy
Proxy Status:            OK, ip 10.244.0.221, 0 redirects active on ports 10000-20000, Envoy: embedded
Global Identity Range:   min 256, max 65535
Hubble:                  Ok              Current/Max Flows: 2233/4095 (54.53%), Flows/s: 7.69   Metrics: Disabled
Encryption:              Disabled        
Cluster health:          2/2 reachable   (2023-10-24T04:23:33Z)
Modules Health:          Stopped(0) Degraded(0) OK(2) Unknown(1)
```

Reply:
Thanks!